libfwupd/fwupd-client.c

```diff
@@ -6398,7 +6398,8 @@ fwupd_client_emulation_load_async(FwupdClient *self,
 	g_dbus_message_set_unix_fd_list(request, fd_list);
 
 	/* call into daemon */
-	g_dbus_message_set_body(request, g_variant_new("(h)", g_unix_input_stream_get_fd(istr)));
+	g_dbus_message_set_body(request,
+				g_variant_new("(sh)", filename, g_unix_input_stream_get_fd(istr)));
```

Comment:
ohh we can't break API like this now -- and also filename will give the security people the shivers as it could contain a path like `../../` -- even if we can show we don't use it.

Reply:
Got it, adjusted it.